packages/connect-webextension/README.md

```diff
@@ -54,6 +54,10 @@ Amend your manifest.json to include the script as a content script. Replace <pat
   ],
 ```
 
+## Adding your webextension to `knownHosts`
+
+To ensure your extension is displayed with its name rather than its ID, you need to open a Pull Request to include it in the knownHosts section of the file located at https://github.com/trezor/trezor-suite/blob/develop/packages/connect/src/data/config.ts#L17.
```

Comment:
`knownHosts`

Reply:
@Hannsek what do you mean by `knownHosts` ? Is there a typo I cannot see?

Reply:
no, it should just be highlighted as a code.